pinning.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// Pinning configures the pinning services.
+type Pinning struct {
+	// Services lists the pinning services
+	Services []PinningServices
+}
+
+type PinningServices struct {
```

Comment:
I guess its singular?
```suggestion
type PinningService struct {
```